### DISCUSSION TITLE
How to migrate Apps from Google Playstore or F-Droid to Obtainium

### DISCUSSION THREAD

**Discussion by rdscorreia74** [ImranR98/Obtainium#1775] [Q&A]
Thanks a bunch for developing Obtainium and making it a FOSS software.
To keep it short, Apps installed via F-Droid have a different signature than on Github/Gitlab. And, for instance, DAVx5 is a real life savior but it doesn't come with a way to export the settings and the accounts. I have maybe a dozen other Apps just like that, no export settings/accounts button...
Is there any way to migrate such Apps to Obtainium while preserving their settings/accounts? If not, what is the "official" way of doing things? Should we simply take good note of all the settings on the App, remove it from the device, install it via Obtainium and then set it up using the notes taken before?
Thanks in advance.
Cheers

**Comment by Mikaela** [comment]
Personally I have ended up to the solution of marking painful to migrate apps as track only and passively following their new releases. Should I notice that an app wasn't getting updated regardless of new releases, then I would have to decide upon further action.

**Comment by derei** [comment]
So, there is no way to force an installation? Can't somehow temporarily disable signature check?

**Reply by DwainZwerg** [reply]
Wouldn't make sense. If you install an app which has nothing to do with the other app on another app, the app could no longer work properly. To exaggerate: You install the Aurora Store via the Google Play Store, because they are both clients for the Google Play Store, so the data and settings are naturally standardized … Of course, they are not standardized.
